lib/index.js

```diff
@@ -167,6 +174,38 @@ I18N.prototype.configure = function( options ) {
 };
 
 I18N.prototype.setLocale = function( localeData ) {
+	if ( localeData && localeData[ '' ] && localeData[ '' ][ 'key-hash' ] ) {
+		var hash, hashLength, keyHash = localeData[ '' ][ 'key-hash' ];
+
+		if ( keyHash.substr( 0, 4 ) === 'sha1' ) {
+			hashLength = Number ( keyHash.substr( 5 ) );
+			hash = function( string ) {
+				var hash = sha1();
+				hash.update( string );
+				hash = hash.digest( 'hex' );
```

Comment:
I can't work out why, but `hash.js` returns a different hash for ellipses. 

Here's a comparison with the [sha1](https://www.npmjs.com/package/sha1) lib.

```
sha1( '…' ) sha1 df316e17de9c36858417ba18e0da4da282dea84a
index.jsx:81 hash.sha1().update('…').digest('hex') hash.js sha1 552cb47dfe2134b226b093b3fa5df3f381984235
```

I've noticed it on the following string:

```
sha1( 'Search all receipts…' ) // 5c4e5bd997b566add1bbc776b546fe6d18f4abe0
hash.sha1().update('Search all receipts…').digest('hex') // 7614de0bfd4652d8f72906c55f0bed4bc5f87b0c
```

<img width="540" alt="screen shot 2018-02-26 at 3 08 04 pm" src="https://user-images.githubusercontent.com/6458278/36654637-27e73dc4-1b12-11e8-9f6a-65d5702d15ff.png">

I haven't tested it with other character entities.

Reply:
That's worrying, thanks for noticing! I have changed the dependency to `sha1` and will do the same for Calypso (which was reason in the first place I opted for hash.js).